### PR TITLE
cmd/allocate: Improvement on error message to make order of args obvious

### PR DESCRIPTION
**Pull request by mattyw**
I'm not totally convinced this is an improvement. But I offer it up here in the name of conversation

### CODE REVIEW

File: cmd/allocate/allocate.go [hunk 1]
```diff
@@ -80,7 +80,7 @@ func (c *allocateCommand) Init(args []string) error {
 	var err error
 	c.Budget, c.Limit, err = parseBudgetWithLimit(budgetWithLimit)
 	if err != nil {
-		return err
+		return errors.Annotate(err, `expected args in the form "service budget:limit"`)
```
Comment (alesstimec) [inline cmd/allocate/allocate.go]: hmm.. yes, i like it.. but maybe we should display the help message if/when something is wrong with arguments..

**Comment by tasdomas** [comment]
I think this is a much better error message.

LGTM

File: cmd/allocate/allocate.go (hunk 1 above)
Comment (alesstimec) [inline cmd/allocate/allocate.go]: and it should be the other way around: 
expected args in the form "budget:limit service"

**Comment by alesstimec** [comment]
LGTM

**Comment by cmars** [comment]
$$merge$$

**Comment by cmars** [comment]
$$merge$$

**Pull request merged**